osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -506,7 +511,6 @@ - (void)showStatus:(enum kaliteStatus)status {
             self.stopButton.enabled = NO;
             self.openBrowserButton.enabled = NO;
             [self.statusItem setToolTip:@"KA-Lite is starting..."];
-            [self.statusItem setImage:[NSImage imageNamed:@"loading"]];
```

Comment:
Nope, that line is at https://github.com/mrpau/installers/blob/feature/276-os-x-application-ux-changes/osx/KA-Lite/KA-Lite/AppDelegate.m#L512 with this code:

> [self.statusItem setImage:[NSImage imageNamed:@"loading"]];